netbox/ipam/api/serializers_/vlans.py

```diff
@@ -21,6 +22,14 @@
 )
 
 
+class NumericRangeArraySerializer(serializers.CharField):
+    def to_internal_value(self, data):
+        return string_to_range_array(data)
+
+    def to_representation(self, instance):
+        return ranges_to_string(instance)
```

Comment:
I'd think we want to represent each range as an item within a list, to enable easy parsing.